### DISCUSSION TITLE
problem after uninstall

### DISCUSSION THREAD

**Discussion by SomeMinecraftModder** [File-New-Project/EarTrumpet#735] [Q&A] [answered]
after i uninstall EarTrumpet, my sound is at 100 and my headphones dont make a lot of sound.

can someone help me ?
(sorry for my bad english)

**Comment by davuxcom** [comment, marked as answer]
Try going to the Windows `Settings app` and then `System` -> `Sound` -> `App volume and device preferences` -> `Reset`.

**Comment by SomeMinecraftModder** [comment]
i try it, dont work.

**Reply by davuxcom** [reply]
At that point I'd suggest reinstalling EarTrumpet and seeing if that fixes it / offers control, and then we can go from there? It sounds like this may not be EarTrumpet. You could also reboot your PC if you have not yet.

**Reply by SomeMinecraftModder** [reply]
I reinstall EarTrumpet, it doesn't work

**Reply by SomeMinecraftModder** [reply]
i reboot its worked! thank you!
